docs/evaluation-metrics.md

```diff
@@ -100,15 +109,55 @@ of elements in the union of the two sets.
 The Levenshtein distance is a measure of similarity between two strings. The Levenshtein distance is calculated as the
 minimum number of insertions, deletions, or substitutions required to transform one string into the other.
 
-### FuzzyWuzzy similarity
+### RapidFuzz similarity
 
 | Configuration Key | Calculation Base     | Possible Values       |
 | ----------------- | -------------------- | --------------------- |
-| `fuzzy`           | `actual`, `expected` | Integer (Fuzzy score) |
+| `fuzzy_score`           | `actual`, `expected` | Integer (Fuzzy score) |
 
-This metric is backed by the [FuzzyWuzzy Python package](https://pypi.org/project/fuzzywuzzy/).
+This metric is backed by the [RapidfFuzz Python package](https://github.com/rapidfuzz/RapidFuzz).
 Calculates the fuzzy score between two documents using the levenshtein distance.
 
+### Rouge retrieval metrics (Token based)
+
+**Rouge** short for Recall-Oriented Understudy for Gisting Evaluation, is typically used in summarization evaluation tasks, comparing human produced references and system generated summaries. But the core idea is to compare and validate sufficient overlap of common words or phrases in both reference and prediction. String metrics look at character level differences, whereas Rouge can help us compare token level matches. We use the [`rouge-score`](https://pypi.org/project/rouge-score/) to compute these measures. Here are some of the metrics we capture.
+
+- **retr_rouge_1_prec**:  
+  - **Description**: ROUGE-1 precision score based on unigram overlap.  
+  - **Interpretation**: This metric measures the precision of unigrams (individual words) between the ground truth context and the retrieved results. Higher precision indicates more relevant retrievals.
+   
+- **retr_rouge_1_recall**:  
+  - **Description**: ROUGE-1 recall score based on unigram overlap.  
+  - **Interpretation**: This metric measures the recall of unigrams between the ground truth context and the retrieved results. Higher recall indicates that more relevant information from the ground truth is captured in the retrievals.  
```

Comment:
You're referring to contexts in these definitions, but I think these metrics are computed using the actual (ground truth answer taken as reference) and expected (generated answer to the question)